### DISCUSSION TITLE
Welcome to TEI Discussions!

### DISCUSSION THREAD

**Discussion by peterstadler** [TEIC/TEI#2120] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PFSchaffner** [comment]
Hi Peter, this apparently happened while I wasn't looking. Is this intended to replace, supplement, or complement TEI-L? Or is that the wrong question?

**Reply by sydb** [reply]
IMHO “supplement” and “compliment” might be a good way to think of this, “replace” would not.

**Reply by peterstadler** [reply]
Wow, @PFSchaffner you've been super fast in noticing :) 
And no, it's not to replace anything, just to provide another space for people to get in touch with the TEI.
